pyvortex/pyproject.toml

```diff
@@ -1,15 +1,20 @@
 [project]
-name = "vortex"
-version = "0.1.0"
-description = "Add your description here"
-authors = [
```

Comment:
If it needs authors, we've started putting `hello@spiraldb.com`

Reply:
it's also easy to create arbitrary mailing list if you need